### PR TITLE
Change API for token prices

### PR DESCRIPTION
**Pull request by dckesler**
## What it solves
Missing prices for some of our tokens
The currency selection was disabled

## How this PR fixes it
Switch to using coingecko's api

## How to test it
Check to make sure CELO, cUSD, and cEUR all have accurate prices.
All available currencies should work with our tokens.

### CODE REVIEW

File: src/logic/currencyValues/api/fetchAvailableCurrencies.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
 
-export const fetchAvailableCurrencies = async (): Promise<GatewayDefinitions['FiatCurrencies']> => {
-  return getFiatCurrencies(getClientGatewayUrl())
+export const fetchAvailableCurrencies = async (): Promise<string[]> => {
+  const geckoCurrencies = (
+    await axios.get<string[]>('https://api.coingecko.com/api/v3/simple/supported_vs_currencies')
+  ).data.map((currency) => currency.toUpperCase())
+  const fiatCurrencies = (
+    await axios.get<{ [currency: string]: string }>('https://openexchangerates.org/api/currencies.json')
+  ).data
```
Comment (aaronmgdr) [inline src/logic/currencyValues/api/fetchAvailableCurrencies.ts:9]: is this doing fetches in a map? 

Suggest a Promise.all for parallel fetching

File: src/logic/safe/api/fetchTokenCurrenciesBalances.ts [hunk 2]
```diff
@@ -109,28 +107,22 @@ async function fetchTokensWithRetriesAndAlternate(address: string) {
...
-  derivedCUSD
-  symbol
-}}`
-
-async function fetchPrices(): Promise<Record<string, number>> {
+async function fetchPrices(tokens: BlockScoutTokenBalance[], selectedCurrency: string): Promise<TokenPrices> {
+  const addressesString = tokens.reduce((addresses, token) => {
+    return `${addresses},${token.contractAddress}`
+  }, '')
+  const priceUrl = `https://api.coingecko.com/api/v3/simple/token_price/celo?vs_currencies=${selectedCurrency}&contract_addresses=${addressesString}`
```
Comment (aaronmgdr) [inline src/logic/safe/api/fetchTokenCurrenciesBalances.ts:120]: hmm this means that coingeck has to either have the version deployed to celo or the contract address on celo must be the same as the one on Eth mainnet (or wherever the token is native to)

File: src/logic/currencyValues/api/fetchAvailableCurrencies.ts (hunk 1 above)
Comment (dckesler) [reply src/logic/currencyValues/api/fetchAvailableCurrencies.ts:9]: This happens just one time on load so I didn't think it was worth it

File: src/logic/safe/api/fetchTokenCurrenciesBalances.ts (hunk 2 above)
Comment (dckesler) [reply src/logic/safe/api/fetchTokenCurrenciesBalances.ts:120]: I do not understand this comment

File: src/logic/currencyValues/api/fetchAvailableCurrencies.ts (hunk 1 above)
Comment (aaronmgdr) [reply src/logic/currencyValues/api/fetchAvailableCurrencies.ts:9]: not worth it. is there a cost?

File: src/logic/safe/api/fetchTokenCurrenciesBalances.ts [hunk 3]
```diff
@@ -58,8 +56,8 @@ async function getTokenBalances(address: string, selectedCurrency: string, exclu
         logoUri: `https://raw.githubusercontent.com/ubeswap/default-token-list/master/assets/asset_${token.symbol}.png`,
       },
       balance: token.balance,
-      fiatBalance: new BigNumber(token.balance).dividedBy(WEI_PER).multipliedBy(fiatConversion).toString(),
-      fiatConversion: fiatConversion.toString(),
+      fiatBalance: new BigNumber(token.balance).dividedBy(WEI_PER).multipliedBy(price).toString(),
+      fiatConversion: '1',
```
Comment (aaronmgdr) [inline src/logic/safe/api/fetchTokenCurrenciesBalances.ts:60]: why always 1? shouldnt this be different for different fiat currencies?

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**